server/athenian/api/align/queries/goals.py

```diff
@@ -244,20 +272,27 @@ def _team_goal_rows_to_request(
                 ),
             )
 
+        time_intervals = [initial_interval, (valid_from, expires_at)]
```

Comment:
```suggestion
        time_intervals = [initial_interval, [valid_from, expires_at]]
```
so that we have lists everywhere

Reply:
we need `Intervals` (tuple) as elements of time_intervals since they will be used as keys in `TeamMetricsResult`

Reply:
OK, so we have tuples everywhere.